src/topic/components/Participants.js

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import PieChart from 'react-svg-piechart';
+import Colors from '../../styles/Colors';
+
+const Participants = props => {
+    const VOTES_NEEDED = 40;
+    const missingVotes = VOTES_NEEDED - props.likes;
+    const chartData = [
+        {title: 'Liczba głosów', value: props.likes, color: Colors.yellow},
+        {title: 'Brakujące Głosy', value: missingVotes, color: Colors.red},
+    ];
+
+    return (<StyledParticipants>
+        <PieChart
+            data={chartData}
+            // If you need expand on hover (or touch) effect
```

Comment:
[opinion/minor] I believe that props are more like self-explandatory - thus no additional comments are necessary